ci/meta.yml

```diff
@@ -24,26 +24,27 @@ meta:
         outputs:
           - name: terraform-config
       params:
-        AWS_ACCESS_KEY_ID: ((ci.aws_access_key_id))
-        AWS_SECRET_ACCESS_KEY: ((ci.aws_secret_access_key))
         AWS_REGION: eu-west-2
-    terraform-apply:
-      task: terraform-apply
+
+    terraform-common-config:
       config:
         platform: linux
         image_resource:
           type: docker-image
           source:
-            repository: ((terraform.repository))
-            version: ((terraform12.version))
-            tag: ((terraform12.version))
+            repository: hashicorp/terraform
+            version: ((dataworks.terraform_version))
+            tag: ((dataworks.terraform_version))
```

Comment:
you don't need `tag` in newer Concourse, just `version

Reply:
for reference: https://concourse-ci.org/tasks.html#schema.anonymous_resource

Reply:
I think we need tag, but not version (from what I can see in the docs)